crypto/src/main/java/org/hyperledger/besu/crypto/SECP256K1.java

```diff
@@ -187,9 +187,17 @@ public static Signature sign(final Bytes32 dataHash, final KeyPair keyPair) {
     signer.init(true, privKey);
 
     final BigInteger[] components = signer.generateSignature(dataHash.toArrayUnsafe());
-    final BigInteger r = components[0];
-    BigInteger s = components[1];
 
+    return normaliseSignature(components[0], components[1], keyPair.getPublicKey(), dataHash);
+  }
+
+  public static Signature normaliseSignature(
+      final BigInteger nativeR,
+      final BigInteger nativeS,
+      final PublicKey publicKey,
+      final Bytes32 dataHash) {
```

Comment:
DataHash shouldn't be needed here

Reply:
its used to ensure the foward-backward signature works